docs/source/rbac/tech-implementation.md

```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://jupyterhub.readthedocs.io/en/stable/rbac/index.html) allows for requesting tokens with specific permissions.
```

Comment:
Sorry, I'm afraid this is still not the right link. It should be an internal link, e.g. `./index.md`